src/adapters/scene_category.js

```diff
@@ -93,8 +93,14 @@ export default class SceneCategory {
   }
 
   handleLayerMarkerClick = e => {
+    // iframe: markers are not clickable
+    if (window.no_ui) {
```

Comment:
Same remark as previously. Not attaching the listeners in the first place would make more sense.